src/components/structures/MatrixChat.js

```diff
@@ -914,6 +915,14 @@ module.exports = React.createClass({
                 }
             }
         });
+
+        const krh = new KeyRequestHandler(cli);
+        cli.on("crypto.roomKeyRequest", (req) => {
```

Comment:
I thought about that. The problem is that the code ends up looking like:

```
new KeyRequestHandler(cli);
```
or possibly:
```
const krh = new KeyRequestHandler(cli);
krh.registerEventListenersOnTheClientYouAlreadyHave();
```

... neither of which make me happy in any way.